test/components/timeAxisTests.ts

```diff
@@ -179,7 +179,79 @@ describe("TimeAxis", () => {
 
     assert.strictEqual(twoTierSize, oneTierSize * 2, "two-tier axis is twice as tall as one-tier axis");
 
+
+    xAxis.axisConfigurations([
+        [
+           {interval: d3.time.day, step: 2, formatter: Plottable.Formatters.time("%a %e")}
+        ],
+    ]);
+
+    var initialTierSize: number = xAxis.height();
+
+    assert.strictEqual(initialTierSize, oneTierSize,
+                      "2-tier time axis should shrink when prezented new configuration with 1 tier");
+
+    svg.remove();
+  });
+
+  it("three tier time axis should be possible", () => {
+
+    var svg = generateSVG();
+    var xScale = new Plottable.Scale.Time();
+    xScale.domain([new Date("2013-03-23 12:00"), new Date("2013-04-03 0:00")]);
+    var xAxis = new Plottable.Axis.Time(xScale, "bottom");
+    xAxis.gutter(0);
+
+    xAxis.renderTo(svg);
+
+    xAxis.axisConfigurations([
+        [
+           {interval: d3.time.day, step: 2, formatter: Plottable.Formatters.time("%a %e")},
+           {interval: d3.time.day, step: 2, formatter: Plottable.Formatters.time("%a %e")},
+        ],
+    ]);
+
+    var twoTierAxisHeight: number = xAxis.height();
+
+    xAxis.axisConfigurations([
+        [
+           {interval: d3.time.day, step: 2, formatter: Plottable.Formatters.time("%a %e")},
+           {interval: d3.time.day, step: 2, formatter: Plottable.Formatters.time("%a %e")},
+           {interval: d3.time.day, step: 2, formatter: Plottable.Formatters.time("%a %e")},
+        ],
+    ]);
+
+    var threeTierAxisHeight: number = xAxis.height();
+
+    assert.strictEqual(threeTierAxisHeight * 2, twoTierAxisHeight * 3, "two-tier axis is twice as tall as one-tier axis");
+
     svg.remove();
+
+  });
+
+  it("many tier Axis.Time should not exceed the drawing area", () => {
+    var svg = generateSVG(400, 50);
+    var xScale = new Plottable.Scale.Time();
+    xScale.domain([new Date("2013-03-23 12:00"), new Date("2013-04-03 0:00")]);
+    var xAxis = new Plottable.Axis.Time(xScale, "bottom");
+
+    var tiersToCreate = 15;
+    var configuration = Array.apply(null, Array(tiersToCreate)).map(() => {
+      return {interval: d3.time.day, step: 2, formatter: Plottable.Formatters.time("%a %e") };
+    });
+    xAxis.axisConfigurations([configuration]);
+
+    xAxis.renderTo(svg);
+
+    var numberOfVisibleTiers = (<any> xAxis)._element.selectAll(".time-axis-tier").filter(function() {
+      return d3.select(this).style("visibility") === "visible";
+    })[0].length;
+
+    assert.notStrictEqual(numberOfVisibleTiers, tiersToCreate,
+      "there should be less than " + tiersToCreate + " visible tiers");
```

Comment:
This is probably the wrong assert for the test.
This test is trying to check that "none of the visible tiers are drawn outside the `Axis`' bounding box", so we should probably write a check that tests that more directly. Maybe something like filtering the visible tiers, computing their bounding box, and making sure it's inside the `Axis`' bounding box?